README.md

```diff
@@ -1,7 +1,151 @@
-
 # object-store-client
```

Comment:
since this is a public multi-module, you could add the badge: `![](https://img.shields.io/github/v/release/hmrc/object-store-client)`